config/initializers/stripe.rb

```diff
@@ -1,6 +1 @@
-Rails.configuration.stripe = {
-  :publishable_key => ENV['STRIPE_PUBLISHABLE_KEY'] || Rails.application.secrets.stripe_publishable_key,
-  :secret_key      => ENV['STRIPE_SECRET_KEY'] || Rails.application.secrets.stripe_secret_key
-}
-
-Stripe.api_key = Rails.configuration.stripe[:secret_key]
+Stripe.api_key = Rails.application.secret.stripe_secret_key
```

Comment:
This works with env variables if you update secrets.yml as I mentioned in another comment.. will that change be in another commit?